WordPress/create-block-theme
# Add "Recommended Plugins" input box

This PR adds a "Recommended Plugins" input to the wp-admin page and editor versions of the plugin.

Closes https://github.com/WordPress/create-block-theme/issues/399.

### Testing Instructions:
1. Open the wp-admin page or the editor sidebar panel
2. Add some text to the recommended plugins input box
3. Export the theme
4. The text should be added to the theme's readme, under a "Recommended Plugins" heading

### Screenshots:

wp-admin:
<img width="827" alt="image" src="https://github.com/WordPress/create-block-theme/assets/1645628/b71e520f-f39a-4249-9758-e373373a015e">

Editor sidebar panel:
<img width="280" alt="image" src="https://github.com/WordPress/create-block-theme/assets/1645628/77511a29-c25d-4daa-9112-9e497bccdbd5">
